.github/workflows/publish.yaml

```diff
@@ -0,0 +1,42 @@
+name: Publish
+
+on:
+  push:
+    # Publish `v*` tags as releases.
+    tags:
+    - v*
+  pull_request:
```

Comment:
Should we do this on pull requests?

Reply:
We may want to consider pushing an image on push to main, but not on every pull request. For now I'm considering tagging only versions we deem releasable, which will be noted by a tag.